aptible/resolve_handle.go

```diff
@@ -6,6 +6,41 @@ import (
 	"github.com/aptible/go-deploy/client/operations"
 )
 
+// Gets environment id associated with a given handle.
+func (c *Client) GetEnvironmentIDFromHandle(handle string) (int64, error) {
+	params := operations.NewGetAccountsParams()
+	resp, err := c.Client.Operations.GetAccounts(params, c.Token)
+	if err != nil {
+		return 0, err
+	}
+
+	num_accts := *resp.Payload.TotalCount
+	per_pg := *resp.Payload.PerPage
+	page := *resp.Payload.CurrentPage
```

Comment:
Should we do nil checks here? I think these pointer dereferences aren't guaranteed to be safe, right?

Reply:
I don't think they're ever nil, but might as well do it to be safe. I'll add that for `GetDatabaseFromHandle()` function, too.